package.json

```diff
@@ -33,8 +33,11 @@
     "watch": "^0.18.0"
   },
   "scripts": {
-    "build": "node tools/makem.js; echo Built at `date`",
-    "watch": "./node_modules/.bin/watch 'npm run build' ./js/"
+    "build": "cd emscripten/artoolkit5/ && emconfigure ./Configure && cd ../../ && node tools/makem.js; echo Built at `date`",
```

Comment:
i will restore the  ` "build": "node tools/makem.js; echo Built at `date`",` as in the fixing-nft branch, because otherwise the npm run build won't works